methods.go

```diff
@@ -21,6 +21,7 @@ type Method string
 type Event string
 
 const (
+	MethodConnConnect      Method = "sql.conn.connect"
```

Comment:
```suggestion
	MethodConnectorConnect Method = "sql.connector.connect"
```

This method belongs to `otConnector`, so `sql.connector.connect` could be a better name.

Reply:
My bad, renamed it.